FGEGraphics/UISystem/UIDropdown.cs

```diff
@@ -0,0 +1,113 @@
+﻿//
+// This file is part of the Frenetic Game Engine, created by Frenetic LLC.
+// This code is Copyright (C) Frenetic LLC under the terms of a strict license.
+// See README.md or LICENSE.txt in the FreneticGameEngine source root for the contents of the license.
+// If neither of these are available, assume that neither you nor anyone other than the copyright holder
+// hold any right or permission to use this software until such time as the official license is identified.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FGEGraphics.UISystem;
+
+/// <summary>Represents an interactable dropdown of different choices.</summary>
+// TODO: Choice search
+// TODO: Arrow key + ENTER navigation
+// TODO: Scroll if exceed max width
+// TODO: Dropdown icon next to placeholder info?
+public class UIDropdown : UIElement
+{
+    /// <summary>A mapping of <see cref="List"/> clickables to their string values.</summary>
```

Comment:
why the `see List`?

Reply:
Oh - the field is named `List`. That... makes sense but is a little awkward given `List` is also a related coreclr class name